examples/smbclient.py

```diff
@@ -101,17 +102,20 @@ def main():
         else:
             smbClient.login(username, password, domain, lmhash, nthash)
 
-        shell = MiniImpacketShell(smbClient)
+        shell = MiniImpacketShell(smbClient, None, options.outputfile)
 
-        if options.file is not None:
-            logging.info("Executing commands from %s" % options.file.name)
-            for line in options.file.readlines():
+        if options.inputfile is not None:
+            logging.info("Executing commands from %s" % options.inputfile.name)
+            for line in options.inputfile.readlines():
                 if line[0] != '#':
                     print("# %s" % line, end=' ')
                     shell.onecmd(line)
                 else:
                     print(line, end=' ')
         else:
+            f = open(options.outputfile, 'a')
```

Comment:
input and output files aren't mutually exclusive. move these changes outside the else block and check options.outputfile to be active in order to avoid crash when not passing parameter

Reply:
@anadrianmanrique - Submitted commit moving the output file logic out of the if/else for the input file